### PR TITLE
updatereadme

### PR DESCRIPTION
**Pull request by samvid2020**
Hey, Vighnesh can you please approve my first PR just for trial's sake :)

### CODE REVIEW

**Comment by samvid2020** [comment]
HackBerries PR 1

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
-# Project management
 
-> Backend API for project management application, where users can create projects and add clients to the projects
+# Project management Application
```
Comment (VighneshManjrekar) [inline README.md]: Project Management Application API

File: README.md [hunk 2]
```diff
@@ -40,13 +41,32 @@ Starter code is already provided in `controllers/main.controllers.js` file.<br>
 ## How to Contribute
 - Create the issue.
 - Fork the Repo.
+- Create a new Branch [git checkout -b "new branch name"]
 - Fix the issue.
 - Generate a Pull Request.
 - Wait for your PR to get merged.
 
+
+## Documentation
```
Comment (VighneshManjrekar) [inline README.md]: Remove this section documentation should explain about the current repo. 
This blog gives generalized idea that is not required.

File: README.md [hunk 3]
```diff
@@ -40,13 +41,32 @@ Starter code is already provided in `controllers/main.controllers.js` file.<br>
...
+
+
 # Technologies to be used:
 - Nodejs
 
 ## Author & maintainer
 
 [Vighnesh Manjrekar](https://github.com/VighneshManjrekar)
+
+## 🚀 About Me
```
Comment (VighneshManjrekar) [inline README.md:56]: GDSC-DMCE

**Review by VighneshManjrekar — CHANGES_REQUESTED** [review]
Hi Samvid, I appreciate your PR please do these changes.

File: README.md (hunk 3 above)
Comment (samvid2020) [reply README.md:56]: Please go through my changes

**Comment by samvid2020** [comment]
Changes done

File: README.md (hunk 3 above)
Comment (VighneshManjrekar) [reply README.md:56]: Only update GDSC-DMCE keep rest of the part as it is

**Pull request merged**